test_resolwe_runtime_utils.py

```diff
@@ -105,11 +110,6 @@ def test_number(self):
     def test_string(self):
         self.assertEqual(progress('0.1'), '{"proc.progress": 0.1}')
 
-    def test_improper_input(self):
```

Comment:
Please, convert this test to test if we get a platform warning for improper input.